Scripts/script-RACopyFrom.yml

```diff
@@ -0,0 +1,54 @@
+commonfields:
+  id: RACopyFrom
+name: RACopyFrom
+fromversion: 2.0.1
+script: |-
+  var files = executeCommand("copy-from", {"file":args.filePath})
```

Comment:
Script should take and forward "using" arg. Pretty quick and worth it for the added power